MyoSimulatorForm/MyoSimulatorForm/MyoSimulatorForm.cs

```diff
@@ -154,35 +347,12 @@ private void playRecordingToolStripMenuItem_Click(object sender, EventArgs e)
             }
         }
 
-        private void loadScriptToolStripMenuItem_Click(object sender, EventArgs e)
-        {
-            OpenFileDialog openFileDialog = new OpenFileDialog();
-            openFileDialog.Title = "Open Script File";
-            openFileDialog.Filter = "TXT files|*.txt";
-            openFileDialog.InitialDirectory = @"C:\";
-            if (getOpenFileDialogResult(openFileDialog) == DialogResult.OK)
-            {
-                scriptPath.Text = openFileDialog.FileName;
-            }
-        }
-
-        private DialogResult getOpenFileDialogResult(OpenFileDialog dialog)
-        {
-            OpenFileDialogResult dialogResult = new OpenFileDialogResult(dialog);
-            System.Threading.Thread dialogThread = new System.Threading.Thread(dialogResult.threadShowDialog);
-            dialogThread.SetApartmentState(System.Threading.ApartmentState.STA);
-            dialogThread.Start();
-            dialogThread.Join();
-
-            return dialogResult.getResult();
-        }
-
         private void startStopRecordingToolStripMenuItem_Click(object sender, EventArgs e)
         {
             // TODO: Add the recording feature.
             if (!currentlyRecording)
             {
-                currentlyRecording = true;
+                currentlyRecording = true; // ***false??
                 startStopRecordingToolStripMenuItem.Text = stopRecordingLabel;
```

Comment:
Sorry, this comment was only for myself when I was first reading through it. It should be true, but I forgot to delete the comment.